### ISSUE TITLE
Support ROX(Read Only Many) PVC as file source.

### ISSUE THREAD

**Issue by mugioka** [grafana/k6-operator#279]
### Feature Description
ROX(Read Only Many) PVC should be supported as they allow load testing distributed across multiple nodes.
Now, k6-operator does not support ROX PVC as file source.
To do so we will need to support ReadOnly parameter in Volumes/VolumeMounts.
https://cloud.google.com/kubernetes-engine/docs/how-to/persistent-volumes/readonlymany-disks
> In your Pod specification, you must specify readOnly: true in the volumeMounts section and the volumes section, such as in the following example:

### Suggested Solution (optional)

```yaml
apiVersion: k6.io/v1alpha1
kind: K6
metadata:
  name: k6
spec:
  script:
    volumeClaim:
      name: scenario-resources
      readOnly: true
```

**Comment by yorugac** [comment]
Hi @mugioka, thanks for the issue!

I can't help but wonder if this (`readOnly: true`) shouldn't be a hard-coded default instead of adding a new field to CRD, esp. now that there are volumes in init containers :thinking:

**Comment by mugioka** [comment]
@yorugac 
Considering init containers, I believe a new field should be added to the CRD to avoid making destructive changes to the k6-operator, since they are likely to modify the mounted volumes.

**Comment by yorugac** [comment]
Closed via #415
